packages/pma/pitop/pma/light_sensor.py

```diff
@@ -15,8 +15,16 @@ class LightSensor(ADCBase):
     :param str port_name: The ID for the port to which this component is connected
     """
 
-    def __init__(self, port_name, pin_number=1, name="light_sensor"):
-        ADCBase.__init__(self, port_name=port_name, pin_number=pin_number, name=name)
+    def __init__(
+        self, port_name, pin_number=1, name="light_sensor", number_of_samples=3
```

Comment:
i think we should mention in the doc description that by default 3 samples are taken so the reading takes around 0.5 seconds

Reply:
makes sense; also just added documentation for the `name` parameter which was missing